src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -229,19 +235,38 @@ protected Map<String, String> create(final SscsCaseDataWrapper responseWrapper,
             personalisation.put(JOINT_PARTY_APPEAL, "No");
         }
 
-        if (ccdResponse.getHearings() != null && !ccdResponse.getHearings().isEmpty()) {
-
-            Hearing latestHearing = NotificationUtils.getLatestHearing(ccdResponse);
-            if (latestHearing != null) {
-                HearingDetails latestHearingValue = latestHearing.getValue();
-                LocalDateTime hearingDateTime = latestHearingValue.getHearingDateTime();
-                personalisation.put(HEARING_DATE, formatLocalDate(hearingDateTime.toLocalDate()));
-                translateToWelshDate(hearingDateTime.toLocalDate(), ccdResponse, value -> personalisation.put(WELSH_HEARING_DATE, value));
-                personalisation.put(HEARING_TIME, formatLocalTime(hearingDateTime));
-                personalisation.put(VENUE_ADDRESS_LITERAL, formatAddress(latestHearing));
-                personalisation.put(VENUE_MAP_LINK_LITERAL, latestHearingValue.getVenue().getGoogleMapLink());
-                personalisation.put(DAYS_TO_HEARING_LITERAL, calculateDaysToHearingText(hearingDateTime.toLocalDate()));
+        Hearing latestHearing = ccdResponse.getLatestHearing();
+
+        HearingRoute hearingRoute = ccdResponse.getSchedulingAndListingFields().getHearingRoute();
+        if (HearingRoute.LIST_ASSIST == hearingRoute) {
+            latestHearing = ofNullable(ccdResponse.getHearings())
+                .orElse(Collections.emptyList()).stream()
```

Comment:
Isn't there a getLatestHearing on sscsCaseData?

Reply:
Yes, I use it above, but the filter on the next line only allows list assist hearings that have been booked by checking epims id.

Reply:
Could you check to see if the latest hearing has an epims id instead of going through all the hearings again?